mobile_verifier/src/heartbeats.rs

```diff
@@ -459,6 +465,7 @@ async fn validate_heartbeat(
     if coverage.max_distance_km(latlng) > max_distance {
         return Ok((cell_type, proto::HeartbeatValidity::TooFarFromCoverage));
     }
+    */
```

Comment:
you said "removed" did you mean more specifically that we _should_ be invalidating these in phase 3 so this is only temporarily commented out or was this intended to be deleted once it successfully compiled or something?